layersvt/VkLayer_screenshot.json.in

```diff
@@ -25,23 +25,23 @@
                     "key": "frames",
                     "env": "VK_SCREENSHOT_FRAMES",
                     "label": "Frames",
-                    "description": "Comma separated list of frames to output as screen shots or a range of frames with a start, count, and optional interval separated by a dash. Setting the variable to \"all\" will output every frame. Example: \"5-8-2\" will output frame 5, continue until frame 13, dumping every other frame. Example: \"3,8-2\" will output frames 3, 8, and 9.",
+                    "description": "Specifies list of frames to output as screenshots. It is specified as a comma separated list of frames or a range of frames with a start, count, and optional interval separated by a dash. Setting the variable to \"all\" will output every frame. Example: \"5-8-2\" will output frame 5, continue until frame 13, dumping every other frame. Example: \"3,8-2\" will output frames 3, 8, and 9.",
                     "type": "STRING",
                     "default": ""
                 },
                 {
                     "key": "dir",
                     "env": "VK_SCREENSHOT_DIR",
                     "label": "Directory",
-                    "description": "This can be set to specify the directory in which to create the screenshot files.",
+                    "description": "Specifies the directory in which to create the screenshot files. If it is not set or is set to null, the files will be created in the current working directory.",
```

Comment:
I don't think you *can* set a variable to `null`, either in an environment variable or a layer settings file, right?  Does this mean "If it is not set or is set to the empty string"?

Also, why does this setting (and others below) not specify `platforms`?  Is it because all platforms are supported by default?  I note that several of the device simulation settings specify:
```
"platforms": [ "WINDOWS", "LINUX", "MACOS", "ANDROID" ], 
```
so I was assuming that all platforms are always listed...